### PR TITLE
fix: PhysicalResourceID appended

### PR DESCRIPTION
**Pull request by komushi**
# Description

## Type of change

- [*] *Bug fix (non-breaking change which fixes an issue)*
- [] *New feature (non-breaking change which adds functionality)*
- [] *Breaking change (fix or feature that would cause existing functionality to not work as expected)*
- [] *Refactor* (existing code being refactored)
- [] *This change requires a documentation update*

# Submission Checklist

- [*] Build Verified
- [*] Bundle Verified
- [*] Lint passing
- [*] Unit tests passing
- [*] Integration tests passing
- [ ] Change logs generated

### CODE REVIEW

**Comment by bahavar** [comment]
Please resolve any remaining conflicts.

File: source/rush.json [hunk 1]
```diff
@@ -531,9 +531,6 @@
      * The default branch name. This tells "rush change" which remote branch to compare against.
      * The default value is "master"
      */
-    /*  INTERNAL REPO:  */
```
Comment (bahavar) [inline source/rush.json:534]: Do not delete these lines

File: source/packages/libraries/core/deployment-helper/src/lambda_custom_resource_proxy.ts [hunk 2]
```diff
@@ -30,7 +32,14 @@ exports.handler = async (event: CustomResourceEvent, context: unknown) => {
 
     try {
         const resourceResult = await customResourceManager[event.RequestType.toLowerCase()](event);
-        return await send(event, context, 'SUCCESS', resourceResult);
+        
+        logger.debug(`existing PhysicalResourceId: ${event.PhysicalResourceId}`);
+        
+        const physicalResourceId = event.PhysicalResourceId || uuid();
+        
+        logger.debug(`Determined PhysicalResourceId:${physicalResourceId}`);
```
[the remark this answers is not in the document]
Comment (bahavar) [reply source/packages/libraries/core/deployment-helper/src/lambda_custom_resource_proxy.ts]: @komushi , please respond

**Comment by boardthatpowder** [comment]
Hi @komushi can you confirm if this is still needed or not? If yes, please resolve conflicts. If no, please delete. Thanks.

**Comment by boardthatpowder** [comment]
Closing this without merging due to inactivity. Please reopen and address outstanding review comments if it's still required.

**Comment by komushi** [comment]
The customer use-case needs to include the uuid library.
We solved the conflicts in the latest commit - c3ba11a008140cca83fd010354a8b0751bd6223c.
It is expected to merge this into main.

**Comment by boardthatpowder** [comment]
Closing due to lack of activity.

**Pull request closed without merging**